examples/typescript/pipe-screen-time-storyteller/pipe.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "Screen Time Storyteller",
+  "version": "1.0.0",
+  "description": "Generates a narrative summary of your day based on your screen activities",
+  "author": "AI Assistant",
+  "fields": [
+    {
+      "name": "aiProvider",
+      "value": "ollama"
+    },
+    {
+      "name": "models",
+      "value": {
```

Comment:
does this work? did you test loading this pipe in the app UI?

it is not supposed to work like this


check example: https://github.com/mediar-ai/screenpipe/blob/main/examples/typescript/pipe-meeting-summary-by-email/pipe.json

default should be present, not value, and it will show as placeholder in app ui inputs

also object type does not exist, not sure how you tested this